.dockerignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+npm-debug.log
```

Comment:
Thanks for your feedback 👍 .The `dist` folder is where the *production* code is outputted to. It will be empty until the `build` command is run, then this is the code that will run on the Docker container.

Reply:
Adding files to `.dockerignore` will not copy them to the docker image. There are chances that there might be a local `dist` folder when a local build is run. Adding `dist` to `.dockerignore` will ensure that this is not copied to the docker image. So even if `dist` is added to `.dockerignore`, a dist folder can be created inside the docker container when the build command is run.

Reply:
I can add it, but as it is built on CI in a new container each time, the `dist` directory will be empty. It is better to be safe 👍